lib/models/versionAndUrl.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:version/version.dart';
+
+class VersionAndUrl {
+  Version? version;
```

Comment:
Should this be nullable?
That would mean, there can be a VersionAndUrl without a Version, so just URL